php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2257,8 +2261,14 @@ abstract class NDB_BVL_Instrument extends NDB_Page
         foreach ($columns as $row) {
             switch ($row['COLUMN_NAME']) {
             case 'CommentID':
+            case 'ID':
+                continue 2;
+            case 'SessionID':
+                continue 2;
             case 'UserID':
                 continue 2;
+            case 'Test_name':
+                continue 2;
```

Comment:
```suggestion
            case 'ID':
            case 'SessionID':
            case 'UserID':
            case 'Test_name':
                continue 2;
```

Reply:
Or, those COLUMN_NAME could be added to the query above in a `NOT IN (...)` condition

Reply:
I prefer the change you suggested